frontend/pages/Homepage/cards/MDM/MDM.tsx

```diff
@@ -84,13 +108,20 @@ const MDM = ({
         },
         { status: "Unenrolled", hosts: unenrolled_hosts_count },
       ]);
+      setSolutions(mobile_device_management_solution);
     },
     onError: () => {
       setShowMDMUI(true);
     },
   });
 
-  const tableHeaders = generateTableHeaders();
+  const onTabChange = (index: number) => {
+    setNavTabIndex(index);
+  };
+
+  const solutionsTableHeaders = generateSolutionsTableHeaders();
```

Comment:
Since the table headers are not changing based on any parameters, I think you can just import the table headers as constants directly from the config file without calling a separate generate function (which I think will be called on each render).